openapi/crm/crm.json

```diff
@@ -0,0 +1,381 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "CRM",
+    "version": "1.0",
+    "description": ""
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/org",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/org",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/org",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/org",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
```

Comment:
can we update the description to reflect that the values will either Contact or Company or Activities. Probably we can enforce an enum in the crm.go for these values.

Reply:
The description is already updated as part of [PR](https://github.com/vendasta/api-gateway-docs/pull/185).